opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -381,8 +385,8 @@ def add_log_record_processor(
     def _submit_and_wait(
         self,
         func: Callable[[LogRecordProcessor], Callable[..., None]],
-        *args: Any,
-        **kwargs: Any,
+        *args: OtelAny,
```

Comment:
I think we only want this typing for the LogRecord body no?

Reply:
I did this change and rename the custom Any to "AnyValue" as you mentioned above.
You can review the changes